ui/app/AppLayouts/Chat/ChatColumn/TopBar.qml

```diff
@@ -7,7 +7,22 @@ import "../../../../imports"
 import "../components"
 
 Rectangle {
-    property bool isModeratedChat: chatsModel.activeChannel.name.startsWith(Constants.moderatedChannelPrefix)
+    property bool isModeratedChat: chatsModel.activeChannel.name.startsWith(Constants.moderatedChannelPrefix);
+
+    property bool isOperator: false
+
+    Connections {
+        target: chatsModel.activeChannel
+        onChatItemChanged: {
```

Comment:
Is this an event when the active chat changes? Is so, there is already on called `onActiveChannelChanged`